src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeFixes/DocumentBasedFixAllProvider.cs

```diff
@@ -9,7 +9,10 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.PooledObjects;
+
+#if NETSTANDARD2_0 || CODE_STYLE
```

Comment:
The code style layer is netstandard2.0, so this condition can be simplified.

```suggestion
#if NETSTANDARD2_0
```